lib/src/allocation.dart

```diff
@@ -35,9 +35,9 @@ typedef WinHeapAlloc = Pointer Function(Pointer, int, int);
 final WinHeapAlloc winHeapAlloc =
     stdlib.lookupFunction<WinHeapAllocNative, WinHeapAlloc>('HeapAlloc');
 
-typedef WinHeapFreeNative = Int32 Function(
+typedef WinHeapFreeNative = Bool Function(
```

Comment:
On windows `BOOL` is a typedef to `int` AFAIK. Does an actual `bool` equal to `int` in calling convention?

Reply:
Yeah, this is incorrect in its changed form. In general, Win32 doesn't use the `bool` type. `BOOL` is defined as follows in `minwindef.h`:

```c
typedef int                 BOOL;
```